front/components/profile/MainTab.tsx

```diff
@@ -30,7 +30,7 @@ export default function MainTab({
       ),
     },
     {
-      name: "DataSources",
+      name: "Data Sources",
```

Comment:
This will break the selection UI in the tab bar if the name does not match in the actual page. Based on the PR I don't think it's changed there. Expected behavior as you click on it it won't show the highlight below it?

Reply:
Correct. Fixed.